### DISCUSSION TITLE
Allow updating the Regex used for the component builder

### DISCUSSION THREAD

**Discussion by mg-aceik** [Sitecore/jss#1730] [Ideas]
### Is your suggestion related to a problem? Please describe.

I want to co-locate component .stories.tsx and .test.tsx files in the same folder under src/components.
To do this and to exclude them from the component builder I need to be able to set this regex.
https://github.com/Sitecore/jss/blob/7905a71067fd07cd21dcd6f9dc1a63abcccd4abb/packages/sitecore-jss-dev-tools/src/templating/utils.ts#L42

### Describe the solution you'd like

Allow passing this regex as a param in the component builder.

### Is there a way to achieve this functionality with exisiting tools?

My current workaround is to copy https://github.com/Sitecore/jss/blob/7905a71067fd07cd21dcd6f9dc1a63abcccd4abb/packages/sitecore-jss-dev-tools/src/templating/nextjs/generate-component-builder.ts into my solution and modify it to remove these unwanted components.

**Comment by art-alexeyenko** [comment]
Good suggestion, thank you @mg-aceik . I've put it in our backlog.
We have an item for a filter functionality that does a similar things, suggested here: https://github.com/Sitecore/jss/discussions/1551 too, so we'll decide which approach is better.
